web/js/landingView.js

```diff
@@ -73,33 +75,44 @@
   };
 
   var addHandlers = function () {
-    enterButton.addEventListener('click', function onEnterClicked(event) {
-      event.preventDefault();
-      event.stopImmediatePropagation();
-
-      var form = document.querySelector('form');
-      if (!isValid()) {
-        form.classList.add('error');
-        return;
-      }
-
-      form.classList.remove('error');
-      enterButton.removeEventListener('click', onEnterClicked);
-
-      if (isWebRTCVersion) {
-        showContract().then(function (accepted) {
-          if (accepted) {
-            navigateToRoom();
-          } else {
-            addHandlers();
-          }
-        });
-      } else {
-        navigateToRoom();
-      }
-    });
+    enterButton.addEventListener('click', onEnterClicked);
+
+    for (var i = 0; i < inputFields.length; i++) {
```

Comment:
The enter to submit form is HTML and shouldn't need all this js: https://www.w3schools.com/html/html_forms.asp